test/config.test1.yaml

```diff
@@ -60,9 +60,11 @@ base_network:
   min_voltage_rebase_voltage: 35000
 
 load_options:
-  weather_year: 2014
-  prediction_year: 2030
-  scale: 1
+  ssp: "ssp2-2.6" # shared socio-economic pathway (GDP and population growth) scenario to consider
+  weather_year: 2018  # Load scenarios available with different weather year (different renewable potentials)
```

Comment:
Hi @giacfalk, just talked at the data workflow meeting with Davide about this.
Fixing this to 2013 is probably the fasted approach.

Can you update your branch on GitHub and test if it works with 2013?
We would welcome the PR :)